mix.exs

```diff
@@ -34,11 +34,12 @@ defmodule MLLP.MixProject do
       {:telemetry, "~> 0.4.3"},
       {:ranch, "~> 1.7.1"},
       {:elixir_hl7, "~> 0.6.0"},
+      {:ex_doc, "~> 0.24.2", only: :dev, runtime: false},
       {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false},
       {:mix_test_watch, "~> 1.0.2", only: :dev, runtime: false},
       {:mox, "~> 1.0.0", only: :test},
       {:excoveralls, "~> 0.14.0", only: :test, runtime: false},
-      {:private, "~> 0.1.1", runtime: false}
+      {:private, "~> 0.1.1", runtime: false},
```

Comment:
`private` dep, TIL. I see you have added its dependency. Are we testing at present private functions.

Reply:
I didn't add  it, it just got moved (i.e, it's in mix.exs in main), but IMO it's not in use, so let's remove it, thoughts?

Reply:
Oh, I didn't see it in main. I am OK removing it